### PR TITLE
Fixed validation on deactivation and withExecution

### PR DESCRIPTION
**Pull request by georgiana-ecobici**
Change-Id: I937f837f31791a2b25f4205987ff9de2da4e6ca0
Signed-off-by: Georgiana-Elena ECOBICI <georgiana-elena.ecobici@thalesgroup.com>

### CODE REVIEW

File: plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend [hunk 1]
```diff
@@ -153,24 +149,34 @@ class TextualScenarioValidator extends AbstractTextualScenarioValidator {
 	 */
 	@Check
 	def checkWithExecutionHasDeactivate(Model model) {
-		var messagesWithExecution = newHashMap
+		// keep a list with the messages that contains the withExecution keyword
```
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend]: update this comment -> the list contains target names, not messages

File: plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend [hunk 2]
```diff
@@ -153,24 +149,34 @@ class TextualScenarioValidator extends AbstractTextualScenarioValidator {
...
+		// keep also a list with the index on which withExecution is found, to know on which line to show an error
+		var messagesWithExecution = newLinkedList
```
Comment (MalinaStefaniaStoicanescu) [inline plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend]: the name of the variable could be updated too

File: plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend (hunk 1 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend]: Done

File: plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend (hunk 2 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend]: Done

**Review by MalinaStefaniaStoicanescu — APPROVED** [review]

**Pull request merged**